File: app/src/main/java/life/plank/juna/zone/data/network/model/FixtureByMatchDay.java
package life.plank.juna.zone.data.network.model;

import java.util.List;

import lombok.Data;

@Data
public class FixtureByMatchDay {
    private String daySection;
    private int matchDay;
    private List<FixtureByDate> fixtureByDateList;

    private FixtureByMatchDay(String daySection, int matchDay, List<FixtureByDate> fixtureByDateList) {
        this.daySection = daySection;
        this.matchDay = matchDay;
        this.fixtureByDateList = fixtureByDateList;
    }
}

File: app/src/main/java/life/plank/juna/zone/data/model/football/FixtureByMatchDay.kt
package life.plank.juna.zone.data.model.football

import android.os.Parcelable
import kotlinx.android.parcel.*
import java.util.*
import kotlin.collections.ArrayList

@Parcelize
data class FixtureByMatchDay(
        var matchDay: Int = 0,
        var daySection: String = "",
        var fixtureByDateList: @RawValue MutableList<FixtureByDate> = Collections.emptyList()
) : Parcelable {
    constructor(matchDay: Int) : this(matchDay, "", ArrayList<FixtureByDate>())
}